scipy/optimize/_minimize.py

```diff
@@ -65,6 +65,11 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
         where ``x`` is a 1-D array with shape (n,) and ``args``
         is a tuple of the fixed parameters needed to completely
         specify the function.
+
+        Suppose the callable has signature ``f0(x, *args, **kwargs)``, where ``args``
+        and ``kwargs`` are required positional and keyword arguments. Rather than
+        passing ``f0`` as the callable, consider wrapping it to accept only ``x``;
```

Comment:
```suggestion
        passing ``f0`` as the callable, wrap it to accept only ``x``;
```
(And in the documentation of `args`, discourage use in new code.)